images/virtualization-artifact/pkg/controller/vm_reconciler.go

```diff
@@ -735,7 +758,26 @@ func (r *VMReconciler) syncPowerState(ctx context.Context, state *VMReconcilerSt
 		// kubevirt restarts VM via re-creation of KVVMI.
 		err = state.EnsureRunStrategy(ctx, virtv1.RunStrategyAlways)
 	case virtv2.AlwaysOnUnlessStoppedManualy:
-		if state.KVVMI != nil && state.KVVMI.DeletionTimestamp == nil {
+		if state.KVVM.Status.PrintableStatus == virtv1.VirtualMachineStatusStopped {
+			_, hasAnno := state.KVVM.GetAnnotations()["virtualization.deckhouse.io/vm.powerstate.start-on-creation"]
+			if hasAnno {
+				// Add start request to start newly created VM.
+				opts.Log.Info("Patch KVVM to delete start-on-creation annotation")
+				patch := `[{"op":"remove", "path": "/metadata/annotations/virtualization.deckhouse.io~1vm.powerstate.start-on-creation"}]`
```

Comment:
we have pkg with [JsonPatch](https://github.com/deckhouse/virtualization/blob/cd942696adfba67ff6da62358922bf27a1cdc6e1/images/virtualization-artifact/pkg/common/patch/patch.go#L14)